src/vivarium_public_health/disease/transition.py

```diff
@@ -28,6 +30,35 @@ def __new__(cls, value):
 
 
 class RateTransition(Transition):
+
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def configuration_defaults(self) -> Dict[str, Any]:
+        return {
+            f"{self.name}": {
+                "data_sources": {
+                    "transition_rate": "self::load_transition_rate",
+                },
+            },
+        }
+
+    @property
+    def transition_rate_pipeline_name(self) -> str:
+        if "incidence_rate" in self._get_data_functions:
+            pipeline_name = f"{self.output_state.state_id}.incidence_rate"
+        elif "remission_rate" in self._get_data_functions:
+            pipeline_name = f"{self.input_state.state_id}.remission_rate"
+        elif "transition_rate" in self._get_data_functions:
+            pipeline_name = (
+                f"{self.input_state.state_id}_to_{self.output_state.state_id}.transition_rate"
+            )
+        else:
+            raise ValueError("Cannot determine rate_transition pipeline name.")
```

Comment:
Is there a way to make this more informative for the user? Maybe even just saying something like "expected to find 'incidence_rate', 'remission_rate', or 'transition_rate' but was provided {list(get_data_functions.keys())}"